README.md

```diff
@@ -32,17 +36,17 @@ Scheduler wall time statistics are now disabled by default to keep in line with
 
 That's because you should use tags for stable versions instead! The changelog below should let you know what to expect.
 
-## CHANGELOG ##
```

Comment:
We're gonna need a 2.0.0 changelog entry though I guess I can write that one up.